alune/resources/config.yaml

```diff
@@ -8,6 +8,10 @@ log_level: "INFO"
 # Please note that changing this may require changing traits as well.
 game_mode: "normal"
 
+# Timeout for the bot to stay in the queue. I've seen some instances where the bot is in a queue for several dozen minutes.
+# This is the maximum time the bot will wait in the queue before it will exit the game. 150 secs seems to be a good value.
```

Comment:
Please leave out personal comments from the config. The following is fine (feel free to change wording, it's a suggestion):
```
# Maximum amount of time in seconds the bot should stay in queue before restarting it.
# This prevents endless queues. Increase this on servers with less player population.
```